### ISSUE TITLE
magic apiary ISided Broken

### ISSUE THREAD

**Issue by dualinfinities** [MagicBees/MagicBees#151]
bottom and top sides access inputs only, allowing extracting from input (!!!), cardinal sides refuse bee input, allow extraction of products.

**Comment by Timelife** [comment]
I looked at the code (I really don't know how to program, but I can understand some things). It appears that the method canExtractItem in class TileEntityMagicApiary doesn't have a case for the queen and drone slots, leading it to default to true. The fact that the queen and drone slots can only be accessed from top and bottom I believe is intentional. I don't know why it is not inserting the queen and drone, from the looks of it it would insert the drone, then extract it if it was an in/out pipe (like ender conduits).

**Comment by Timelife** [comment]
Hmmmm, I managed to get a setup that works for automation, just make sure the top is insert only, works with 2 ender io item conduits.

**Comment by dualinfinities** [comment]
yeah, but it's very awkward, especially since the forestry apiaries don't
work that way at all.
